utils/utils.h

```diff
@@ -1,11 +1,13 @@
 #ifndef _UTILS_H_
 #define _UTILS_H_
+#include <glm/glm.hpp>
```

Comment:
Is this needed?

Reply:
i think not anymore since i moved the math helpers out